datapackage.json

```diff
@@ -1,118 +1,122 @@
 {
-  "name": "s-and-p-500-companies-financials",
-  "title": "S&P 500 Companies with Financial Information",
-  "licenses": [
```

Comment:
don't delete liceses

Reply:
They weren't deleted, merely replaced with a shorter line. I restored the original longer version now